homeassistant/components/airq/config_flow.py

```diff
@@ -72,3 +74,40 @@ async def async_step_user(
         return self.async_show_form(
             step_id="user", data_schema=STEP_USER_DATA_SCHEMA, errors=errors
         )
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> config_entries.OptionsFlow:
+        """Return the options flow."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlowWithConfigEntry):
+    """Handle options flow for air-Q."""
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Manage the options."""
+        if user_input is not None:
+            self.options.update(user_input)
+            return self.async_create_entry(title="", data=self.options)
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
```

Comment:
You can utilize the `homeassistant.helpers.schema_config_entry_flow.SchemaOptionsFlowHandler` class for these simple schema flows.
An example of how that can be implemented can be found here <https://github.com/home-assistant/core/blob/dev/homeassistant/components/traccar_server/config_flow.py#L196-L202>

Reply:
Brilliant, thanks!